contracts/script/EigenDADeployer.s.sol

```diff
@@ -149,6 +155,10 @@ contract EigenDADeployer is DeployOpenEigenLayer {
             address(new TransparentUpgradeableProxy(address(emptyContract), address(eigenDAProxyAdmin), ""))
         );
 
+        eigenDACertVerifierRouter = EigenDACertVerifierRouter(
+            address(new TransparentUpgradeableProxy(address(emptyContract), address(eigenDAProxyAdmin), ""))
```

Comment:
we talked about this on slack that deploying an emptyContract and then upgrading is not necessary. Did you want to change here or you think better to follow the general pattern that all other contracts in this deploy script use?

Reply:
this is for inabox deployments so not necessary